BASE/src/Microsoft.ApplicationInsights/Extensibility/Implementation/Tracing/CoreEventSource.cs

```diff
@@ -587,6 +587,15 @@ public void InitializationIsSkippedForSampledItem(string appDomainName = "Incorr
         [Event(57, Message = "TelemetryChannel found a telemetry item without an InstrumentationKey. This is a required field and must be set in either your config file or at application startup.", Level = EventLevel.Error, Keywords = Keywords.UserActionable)]
         public void TelemetryChannelNoInstrumentationKey(string appDomainName = "Incorrect") => this.WriteEvent(57, this.nameProvider.Name);
 
+        [Event(58, Message = "TelemetryClient.Flush was invoked.", Level = EventLevel.Verbose)]
```

Comment:
Any failures we can log, or they are logged already?

Reply:
I haven't had time to do a thorough investigation to see where failures are caught.